src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioRazorParser.cs

```diff
@@ -77,15 +77,15 @@ internal DefaultVisualStudioRazorParser(RazorCodeDocument codeDocument)
             }
 
             _dispatcher = dispatcher;
-            _templateEngineFactory = templateEngineFactory;
+            _projectEngineFactory = projectEngineFactory;
             _errorReporter = errorReporter;
             _completionBroker = completionBroker;
             _documentTracker = documentTracker;
 
             _documentTracker.ContextChanged += DocumentTracker_ContextChanged;
         }
 
-        public override RazorTemplateEngine TemplateEngine => _templateEngine;
+        public override RazorProjectEngine ProjectEngine => _projectEngine;
```

Comment:
This is a breaking API change but no one consumes this any more.

Reply:
What about in 15.7?

Reply:
Can we delete this or make it internal if no one is using it?